client/api/go.sum

```diff
@@ -0,0 +1,26 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
this file is automatically generated?

Reply:
Yes. It's the go mod's lock file, automatically generated when using `GO111MODULE=on`